pertpy/tools/_metadata.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import anndata
+import pandas as pd
+from scanpy import settings
+
+from pertpy.data._dataloader import _download
+
+
+class MetaData:
+    """Utilities to fetch cell or perturbation metadata."""
+
+    def __init__(self):
+        """Download cell line metadata from DepMap"""
+        cell_line_file_name = "sample_info.csv"
+        cell_line_file_path = settings.cachedir.__str__() + "/" + cell_line_file_name
+        if not Path(cell_line_file_path).exists():
+            print("No DepMap metadata file found. Start downloading now.")
+            _download(
+                url="https://ndownloader.figshare.com/files/35020903",
+                output_file_name=cell_line_file_name,
+                output_path=settings.cachedir,
+                is_zip=False,
+            )
+            self.cell_line_meta = pd.read_csv(settings.cachedir.__str__() + "/" + cell_line_file_name)
+        else:
+            self.cell_line_meta = pd.read_csv(cell_line_file_path)
+
+    def annotate_cell_lines(
+        self,
+        adata: anndata,
+        cell_line_column: str = "DepMap_ID",
+        cell_line_type: str = "DepMap_ID",
```

Comment:
Great implementation! Just a few quips:
- I'm reaaaally picky about parameter names here. I think `cell_line_column` should be `cell_line_identifier` and `cell_line_type` should be `identifier_type`